### ISSUE TITLE
resizeMode and loop props are ignored

### ISSUE THREAD

**Issue by Ilario17** [cornedor/react-native-video-player#68]
React Native 0.49

```
<VideoPlayer 
   videoWidth={win.width - 5}
   videoHeight={win.height - 5} 
   video={{ uri: this.state.video.path }}
   autoplay={false}
   pauseOnPress
   fullScreenOnLongPress={false}
   loop={false}
   resizeMode='cover'
/>
```

In this case: 
`loop={false}` is ignored. When the video end, it starts immediately again
`resizeMode='cover'` is ignored. Seems that the default 'contain' persist

**Comment by cornedor** [comment]
Sorry for the late reaction, had a few busy weeks.
Did you give the player a height?

**Comment by Ilario17** [comment]
Ok, I did some tests. 

Adding `width={win.width}` resolve `resizeMode='cover'`
But seems that the width props is ignored and the default width (200) that you set in this library is not changed.

Changing the line 274 in index.js 
from:
`const { width } = this.state`
to 
`const { width } = this.props`

the size changes according to the value entered.

For the second issue `loop={false}` seems that there are problem when `endWithThumbnail` is not set, because inside the IF at line 173
`if (this.props.endWithThumbnail)`
is set the value `isStarted` to `false`
But if `this.props.endWithThumbnail` is `false` , the `isStarted` is never updated, and the loop doesn't stop.
I remove this if and seems to work well, but I didn't try it with a thumbnail

**Comment by Bigdragon13th** [comment]
I got the same issue. `loop={false}` is not working.

**Comment by cornedor** [comment]
This has been fixed by @mattvot in v0.9.1
